rainfall/parts/query.html

```diff
@@ -1,35 +1,51 @@
-<!-- /wp:column -->
+                        <!-- wp:column {"verticalAlignment":"center","width":"53.34%"} -->
+                        <div class="wp-block-column is-vertically-aligned-center" style="flex-basis:53.34%">
+                            <!-- wp:post-title {"isLink":true,"style":{"typography":{"lineHeight":"1.1","textTransform":"uppercase","fontStyle":"normal","fontWeight":"400"},"color":{"text":"#ffffff","link":"#ffffff"}},"fontSize":"large"} /-->
+                        </div>
+                        <!-- /wp:column -->
 
-<!-- wp:column {"width":"180px"} -->
-<div class="wp-block-column" style="flex-basis:180px"><!-- wp:post-featured-image {"width":"179px","height":"100px"} /--></div>
-<!-- /wp:column -->
+                        <!-- wp:column {"width":"180px"} -->
+                        <div class="wp-block-column" style="flex-basis:180px">
+                            <!-- wp:post-featured-image {"width":"179px","height":"100px"} /-->
+                        </div>
+                        <!-- /wp:column -->
+                    </div>
+                    <!-- /wp:columns -->
+                <!-- /wp:post-template -->
 
-</div>
-<!-- /wp:columns -->
-<!-- /wp:post-template -->
-
-<!-- wp:query-pagination {"layout":{"type":"flex","justifyContent":"space-between"}} -->
-<!-- wp:query-pagination-previous /-->
+                <!-- wp:query-pagination {"layout":{"type":"flex","justifyContent":"space-between"}} -->
+                    <!-- wp:query-pagination-previous /-->
 
-<!-- wp:query-pagination-next /-->
-<!-- /wp:query-pagination --></div>
-<!-- /wp:query --></div>
-<!-- /wp:column -->
+                    <!-- wp:query-pagination-next /-->
+                <!-- /wp:query-pagination -->
+            </div>
+            <!-- /wp:query -->
+        </div>
+        <!-- /wp:column -->
 
-<!-- wp:column {"width":"30%","style":{"spacing":{"padding":{"top":"var:preset|spacing|40","right":"var:preset|spacing|40","bottom":"var:preset|spacing|40","left":"var:preset|spacing|40"}}}} -->
-<div class="wp-block-column" style="padding-top:var(--wp--preset--spacing--40);padding-right:var(--wp--preset--spacing--40);padding-bottom:var(--wp--preset--spacing--40);padding-left:var(--wp--preset--spacing--40);flex-basis:30%"><!-- wp:template-part {"slug":"sidebar"} /--></div>
-<!-- /wp:column --></div>
-<!-- /wp:columns --></div>
+        <!-- wp:column {"width":"30%","style":{"spacing":{"padding":{"top":"var:preset|spacing|40","right":"var:preset|spacing|40","bottom":"var:preset|spacing|40","left":"var:preset|spacing|40"}}}} -->
+        <div class="wp-block-column" style="padding-top:var(--wp--preset--spacing--40);padding-right:var(--wp--preset--spacing--40);padding-bottom:var(--wp--preset--spacing--40);padding-left:var(--wp--preset--spacing--40);flex-basis:30%">
+            <!-- wp:template-part {"slug":"sidebar"} /-->
+        </div>
+        <!-- /wp:column -->
+    </div>
+    <!-- /wp:columns -->
+</div>
```

Comment:
These changes are just indenting the markup, so I could more easily see the differences between this file and home-query..